backend/howtheyvote/worker/__init__.py

```diff
@@ -24,54 +26,82 @@
     Weekday,
     Worker,
     last_pipeline_run_checksum,
-    pipeline_ran_successfully,
 )
 
 log = get_logger(__name__)
 
 
-def op_rcv_midday() -> PipelineResult:
+def rcv_list_handler() -> PipelineResult:
     """Checks if there is a current plenary session and, if yes, fetches the latest roll-call
     vote results."""
     today = datetime.date.today()
 
     if not _is_session_day(today):
         raise SkipPipeline()
```

Comment:
I just though that maybe we want to extract this check into the `schedule_pipeline` function? We have that in multiple places, basically everything that depends on plenary information. Would reduce redundancy, but then we would need a flag that enables/disables the check e.g., for the `MemberPipeline` - so not too sure how much we win there. 
No hard opinion here, just a thought.

Reply:
On one hand, we do indeed have quite a few pipelines with that check. On the other hand, the worker implementation is currently not concerned with details of our specific use case, and I still have hope that one day we’ll find something off-the-shelf that can replace our semi-custom worker… :)